packages/gamut-labs/src/ScoreSummary/index.tsx

```diff
@@ -63,8 +66,26 @@ const renderSubScores = ({
           px={16}
           py={8}
           justifyContent="space-between"
+          alignItems="center"
         >
-          <Text fontWeight="bold">{subContentTitle}</Text>
+          <FlexBox alignItems="center">
+            {colorfulIcons ? (
+              subContentPercentCorrect > 0.7 ? (
+                <Text
+                  height="1rem"
+                  mr={12}
+                  color={platformColors.mint['500'] as Colors}
+                >
+                  <CheckFilledIcon />
+                </Text>
+              ) : (
+                <Text height="1rem" color={'#FA4359' as Colors} mr={12}>
+                  <DeleteFilledIcon />
+                </Text>
+              )
+            ) : null}
+            <Text fontWeight="bold">{subContentTitle}</Text>
+          </FlexBox>
           <FlexBox fontSize={14} minWidth="11rem" justifyContent="flex-end">
             {subContentPercentCorrect <= 0.6 && trackSlug && (
```

Comment:
I think this should be:
```suggestion
            {subContentPercentCorrect < 0.6 && trackSlug && (
```

I went and double-checked from the MVP bug bash doc: https://www.notion.so/codecademy/Summary-Screen-82b2f55558ab4272901343a3ff27a7b5, would you mind fixing while you're here in this file?

Having this threshold be 60% though and the other as 70%, I wonder if we could make them descriptive constants for better readability here. (I originally thought the 60% was a typo and had to go check, and I'd imagine someone else might wonder about it too.) it could be something along the lines of `SHOW_REVIEW_SCORE = 0.6` and `PASSING_SCORE = 0.7`, what do you think?

Reply:
good idea, thanks!